www/src/components/guidelines/color/modal.js

```diff
@@ -0,0 +1,213 @@
+                >
+                  {inverted ? `white` : `black`}Fade[{color}]
+                </Text>
+                {` `}
+                <Flex>
+                  <Text
+                    color={colors[color]}
+                    mr={4}
+                    textAlign="right"
+                    width="40px"
+                  >
+                    {parseFloat(
+                      Math.round(a11yInfo.contrast * 100) / 100
+                    ).toFixed(2)}
+                  </Text>
+                  {a11yLabel}
+                </Flex>
+              </Flex>
+            </Box>
+          )
+        } else {
+          return false
+        }
+      })}
+    </>
+  )
+}
+
+const modalContent = (palette, color) => {
+  let colors = []
+
+  Object.keys(palette[color].colors)
+    .sort((a, b) => b - a)
+    .forEach((colorNumber, index) => {
+      const c = palette[color].colors[colorNumber]
+      const textColor = getTextColor(c.contrast)
+
+      colors.push(
+        <Box width="100%" display={{ lg: `flex` }} key={`palette-${index}`}>
+          <Column bg={c.hex} color={textColor}>
+            <Text
+              as="span"
+              fontWeight={1}
+              color={textColor}
+              fontFamily="monospace"
+              mb={4}
+            >
+              colors.{color}[{colorNumber}] {c.name && c.name}
+            </Text>
+            <Text as="span" ml="auto" mr={4} color={textColor}>
+              {c.hex}
+            </Text>
+            <Text as="span" color={textColor}>
+              {c.rgb.red}, {c.rgb.green}, {c.rgb.blue}
+            </Text>
+            <Box width="100%" flexShrink={0}>
+              <TextSamples
+                contrast={c.contrast}
+                bg={c.hex}
+                colorName={palette[color]}
+              />
+            </Box>
+          </Column>
+          <Column bg="white">
+            <Text as="p" fontFamily="monospace" fontWeight={1} mr={3} mb={4}>
+              colors.{color}[{colorNumber}] {c.name && c.name}
+            </Text>
+            <Text as="span" fontWeight={0} ml="auto">
+              {c.contrast.colorOnWhite.toFixed(2)} /{` `}
+              {getA11yLabel(c.a11y)}
+            </Text>
+            <Box width="100%" flexShrink={0} css={{ "& p": { color: c.hex } }}>
+              {getA11yLabel(c.a11y) === `AA` && <AADescription />}
+              {getA11yLabel(c.a11y) === `AAA` && <AAADescription />}
+              {getA11yLabel(c.a11y) === `AA Large` && <AALargeDescription />}
+            </Box>
+          </Column>
+        </Box>
+      )
+    })
+
+  return colors
+}
+
+const ColorModal = ({ palette, color, handleModalClose }) => {
+  const base = palette[color].colors[palette[color].base]
+
+  return (
+    <>
+      <Flex alignItems="baseline" p={{ xxs: 6, md: 8 }}>
+        <Heading mr={4} mt={0}>
+          {palette[color].name}
+        </Heading>
+        <CloseButton onClick={handleModalClose}>&times;</CloseButton>
```

Comment:
This one needs a label, something like `aria-label={`close ${palette[color].name} modal`}` could work.

Reply:
🤗 👍 Thank you (again) for the review, Marcy!
I think at least the simpler things (like this) are starting to persist! 🤞 🙏  

Fixed in 4bca293